packages/backend/src/services/UserAttributesService/UserAttributesService.ts

```diff
@@ -93,4 +93,41 @@ export class UserAttributesService {
         }
         await this.userAttributesModel.delete(orgAttributeUuid);
     }
+
+    static replaceUserAttributes(
```

Comment:
I think this code should live closer to the sql generation than with the user attribute service code. Although it's business logic, it's function is much more closely related to the `queryBulider` than it is to event tracking, loading data, etc.